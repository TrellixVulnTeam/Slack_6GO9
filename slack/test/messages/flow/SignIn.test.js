const { SignIn } = require('../../../lib/messages/flow');

describe('SignIn rendering', () => {
  test('works', async () => {
    const message = new SignIn('https://example.org/github/oauth/login?state=abcdef');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
