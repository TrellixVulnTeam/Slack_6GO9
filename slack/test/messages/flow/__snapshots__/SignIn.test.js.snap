// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "actions": Array [
        Object {
          "style": "primary",
          "text": "Connect GitHub account",
          "type": "button",
          "url": "https://example.org/github/oauth/login?state=abcdef",
        },
      ],
      "color": "#24292f",
      "fallback": "Finish connecting your GitHub account",
      "text": "Finish connecting your GitHub account",
    },
  ],
  "response_type": "ephemeral",
}
`;
